lib/agent0/agent0/hyperdrive/policies/arbitrage.py

```diff
@@ -0,0 +1,189 @@
+"""Agent policy for smart short positions"""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from agent0.hyperdrive.state import HyperdriveActionType, HyperdriveMarketAction
+from elfpy.types import MarketType, Trade
+from fixedpointmath import FixedPoint
+
+from .hyperdrive_policy import HyperdrivePolicy
+
+if TYPE_CHECKING:
+    from agent0.hyperdrive.state import HyperdriveWallet
+    from ethpy.hyperdrive import HyperdriveInterface
+    from numpy.random._generator import Generator as NumpyGenerator
+
+
+class ArbitragePolicy(HyperdrivePolicy):
+    """Agent that arbitrages based on the fixed rate
+
+    .. note::
+        My strategy:
+            - I arbitrage the fixed rate percentage based on thresholds
+            - I always close any matured open positions
+            - If the fixed rate is higher than `high_fixed_rate_thresh`,
+                I close all open shorts an open a new long for `trade_amount` base
+            - If the fixed rate is lower than `low_fixed_rate_thresh`,
+                I close all open longs and open a new short for `trade_amount` bonds
+    """
+
+    # Lots of arguments in arbitrage bot
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        budget: FixedPoint,
+        rng: NumpyGenerator | None = None,
+        slippage_tolerance: FixedPoint | None = None,
+        trade_amount: FixedPoint | None = None,
+        high_fixed_rate_thresh: FixedPoint | None = None,
+        low_fixed_rate_thresh: FixedPoint | None = None,
+    ):
+        """Initializes the bot
+
+        Arguments
+        ---------
+        budget: FixedPoint
+            The budget of this policy
+        rng: NumpyGenerator | None
+            Random number generator
+        slippage_tolerance: FixedPoint | None
+            Slippage tolerance of trades
+        trade_amount: FixedPoint | None
+            The static amount to trade when opening a position
+        high_fixed_rate_thresh: FixedPoint | None
+            The upper threshold of the fixed rate to open a position
+        low_fixed_rate_thresh: FixedPoint | None
+            The lower threshold of the fixed rate to open a position
+        """
+
+        # Defaults
+        if trade_amount is None:
+            trade_amount = FixedPoint(100)
+        if high_fixed_rate_thresh is None:
```

Comment:
make these strings, we're going to stop supporting floats in the constructor soon